.github/workflows/group_1.yml

```diff
@@ -1,5 +1,5 @@
 name: Group 1
-# Name 1, Name 2 Name 3, ...
+#Travis Barry
```

Comment:
@tbarry2 to match your reports file, make this edit on group_5.yml, you can do another pr and i can close this one.